features/step_definitions/metadata.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const request = require('../support/request');
+const TIMEOUT = 240 * 1000;
+
+module.exports = function server() {
+    // eslint-disable-next-line new-cap
+    this.Before({
+        tags: ['@metadata']
+    }, () => {
+        this.repoOrg = '';
+        this.repoName = 'functional-metadata';
+        this.pipelineId = null;
+        this.eventId = null;
+        this.meta = null;
+        this.jwt = null;
+    });
+
+    this.Given(/^a metadata starts with an empty object$/);
+
+    this.Given(/^an existing pipeline with the workflow:$/, { timeout: TIMEOUT }, table =>
```

Comment:
This is already here: https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/events.js#L21-L53

It will match already.